packages/liquidator/src/liquidator.js

```diff
@@ -237,7 +237,9 @@ class Liquidator {
       at: "Liquidator",
       message: "Checking for under collateralized positions",
       liquidatorOverridePrice: liquidatorOverridePrice ? liquidatorOverridePrice.toString() : null,
-      latestCumulativeFundingRateMultiplier: this.financialContractClient.getLatestCumulativeFundingRateMultiplier(),
+      latestCumulativeFundingRateMultiplier: this.financialContractClient
```

Comment:
This `.toString()` is added becuse it is currently logging the multiplier as a hex string

Reply:
good catch